bin/rock-webapp

```diff
@@ -1,114 +1,3 @@
-        cors_hosts = host
-    end
-    opt.on('--enable-syskit',"load syskit support") do
-        syskit=true
-    end
-    opt.on('--config-dir dir', String, 'directory where the configuration can be found') do |dir|
-        config_dir = dir
-    end
-end
-options.parse ARGV
-
-# set up websocket handling
-Faye::WebSocket.load_adapter('thin')
-thin = Rack::Handler.get('thin')
-EM.next_tick { Rock::WebApp::Tasks.install_event_loop }
-
-if syskit
-    #includes roby app that changes a log path, must be included before Orocos.initialize
-    require 'rock/webapp/syskit'
-end
-
-Orocos::CORBA.name_service.ip = name_server_host
-Orocos.initialize
-
-app = Rack::Builder.new do
-    # create a sprockets environment, and allow plugins to add paths
-    # to it
-    environment = Sprockets::Environment.new
-    environment.append_path File.join( src_dir, 'assets', 'js')
-    environment.append_path File.join( src_dir, 'assets', 'css')
-
-    # load the ruby files in the plugin dir, and call instance_eval
-    # in this way, the plugins can register mappings using the same syntax
-    if config_dir 
-        plugin_dir = File.expand_path( File.join( config_dir, "plugins" ) )
-        if File.directory? plugin_dir
-            # for that, add the directory to the ruby load path
-            $:.unshift( plugin_dir ) unless $:.include? plugin_dir 
-            Dir.glob File.join( plugin_dir, "*.rb" ) do |file|
-                instance_eval File.read( file )
-            end
-        end
-    end
-
-    #map asset handler, which will process the local js files and serve
-    #the client side js code. For now, this will always use sprockets
-    #to serve the assets. In a production system the pipeline would be 
-    #to use a rake task to compile to a static directory, and serve 
-    #statically
-    map '/assets' do
-        run environment
-    end
-    
-    #map the apis
-    map '/api/tasks' do
-        if !cors_hosts.empty?
-            require 'rack/cors'
-            use Rack::Cors do
-                allow do
-                    origins(*cors_hosts)
-                    resource '*', headers: :any, methods: [:get, :post]
-                end
-            end
-        end
-        run Rock::WebApp::Tasks::Root
-    end
-    
-    if syskit
-        map '/api/syskit' do
-            if !cors_hosts.empty?
-                require 'rack/cors'
-                use Rack::Cors do
-                    allow do
-                        origins(*cors_hosts)
-                        resource '*', headers: :any, methods: [:get, :post]
-                    end
-                end
-            end
-            run Rock::WebApp::Syskit::Root
-        end
-    end
-end
-thin.run(app, Host: thin_host, Port: thin_port)
-
+puts "'rock-webapp' is deprecated, please use 'rock-rest-api'"
```

Comment:
Either run rock-rest-api or exit(1). Otherwise, people that start the API as a service will not even get an error. I would personally prefer the first one (exec rock-rest-api) to provide a smoother transition

Moreover, send the message to STDERR.